utils/src/main/java/io/neow3j/contract/NeoURI.java

```diff
@@ -35,6 +47,28 @@ public URI getUri() {
         return this.uri;
     }
 
+    public ScriptHash getAddress() {
+        return ScriptHash.fromAddress(this.address);
```

Comment:
what if the provided address is not a valid Neo address? Exception? or how does it behave?

Also, do you have a test for it? :-)

Reply:
`this.address` can only be set within the builder, and is checked with `AddressUtils.isValidAddress(this.address)` on line 192.